.circleci/config.yml

```diff
@@ -136,10 +136,8 @@ memcached-container: &memcached-container
 redis-container: &redis-container
   image: redis:4.0-alpine
 
-# https://github.com/poseidon/dnsmasq
-# TODO: create our own image like https://www.redhat.com/en/blog/build-ubi-containers-github-actions-buildah-and-podman
 dnsmasq-container: &dnsmasq-container
-  image: quay.io/poseidon/dnsmasq
+  image: quay.io/jlledo/dnsmasq
```

Comment:
`jlledo` is not better than `whoever` :)
To be "our own" it has to be in maybe 3scale-labs account or something.

Reply:
This PR is still a draft, not intended to be reviewed, only to run tests and verify the image works. The repo is at https://github.com/3scale/dnsmasq and I had to upload the image to `quay.io/jlledo` because I don't have the permissions for `quay.io/3scale`.